src/pretix/control/forms/event.py

```diff
@@ -854,6 +854,15 @@ class DisplaySettingsForm(SettingsForm):
         label=_("Show variations of a product expanded by default"),
         required=False
     )
+    frontpage_subevent_ordering = forms.ChoiceField(
+        label=_('Subevent ordering'),
+        choices=[
+            ('date_ascending', _('Event start time')),
+            ('date_descending', _('Event start time (descending)')),
+            ('name_ascending', _('Name')),
+            ('name_descending', _('Name (descending)')),
+        ],
+    )
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
```

Comment:
Can we drop out the field if the event doesn't use subevents? e.g. something like
```
if not self.event.has_subevents:
    del self.fields['frontpage_subevent_ordering']
```
here in the ``__init__`` method after the super call

Reply:
`kwargs['event']` raises a KeyError for me, although I see `self.event = kwargs.pop('event')` used throughout the file. `self.obj` gives me the event though.